executor/src/witgen/machines/fixed_lookup_machine.rs

```diff
@@ -339,6 +316,51 @@ impl<T: FieldElement> FixedLookup<T> {
     }
 }
 
+impl<'a, T: FieldElement> Machine<'a, T> for FixedLookup<'a, T> {
+    fn name(&self) -> &str {
+        "FixedLookup"
+    }
+
+    fn degree(&self) -> powdr_number::DegreeType {
+        // TODO: This function is never actually called by the outside (some machines
+        // are using it in their own implementation though). Also, the FixedLookup machine
+        // is weird, because it is actually responsible for lookups into many different fixed
+        // columns which might have different lengths. That should probably also change in the future.
+        panic!("The FixedLookup machine might not have a consistent degree!")
+    }
```

Comment:
Should we remove this function in this PR then?

Reply:
Thought about it, but once we do split up the fixed lookup into several machines we can implement this again. And I think it's reasonable that one can query the machine height from the outside, maybe it will be useful at some point...

I'm also open to remove it, but then I think it can be in a separate PR.

Reply:
I see your point but to the extent that:

- the fixed lookup machine breaks the Machine interface by not having a single degree
- the degree function is not used

The function should be removed in this PR.

Reply:
Done